settings/src/components/email-address.js

```diff
@@ -18,23 +14,24 @@ import { GlobalContext } from '../script';
  * Render the Email setting.
  */
 export default function EmailAddress() {
-	const { userRecord }                                           = useContext( GlobalContext );
+	const { userRecord } = useContext( GlobalContext );
 	const { record, edit, save, editedRecord, hasEdits, isSaving } = userRecord;
-	const [ emailError, setEmailError ]                            = useState( '' );
-	const [ justChangedEmail, setJustChangedEmail ]                = useState( false );
+	const [ emailError, setEmailError ] = useState( '' );
+	const [ justChangedEmail, setJustChangedEmail ] = useState( false );
```

Comment:
Is it possible for us to avoid these whitespace changes? 

I get that JS and PHP coding standards don't match, but aligning variable assignments makes for so much more readable code..

Reply:
Sure I'll see if we can enforce that

Reply:
It seems that aligning variable assignments isn't possible with Prettier. Closest issue I've found is https://github.com/prettier/prettier/issues/1622

Personally I'm used to this formatting, but I know it irks others. If we feel really strongly we could use prettier to get things in the shape we want in this PR, then disable it. Essentially the current state, but I'll revert these particular whitespace changes.

Reply:
Discussed elsewhere: the team favours following Gutenberg standards and keeping Prettier enabled.